jenkins/README.md

```diff
@@ -14,7 +14,7 @@ publicly visible will get admin rights on the CI jobs. This means :
 * They can start the jobs on their PR directly
 * They can start the jobs for PR of authors that are not in the organization
 * They can add authors to whitelist so that the authors can start jobs on any
-   further PR on their own, by commenting **add to whitelist** on the PR
+   further PR on their own, by commenting **/ok-to-test** on the PR
```

Comment:
This doesn't seem correct to me. Adding /ok-to-test doesn't allow the person to start jobs on the further PRs

Reply:
Sure, clarified it a bit more